CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-- Fix: default express limit to 1Mb instead default 100Kb and allow change it throught a conf env var 'IOTA_EXPRESS_LIMIT' (telefonicaid/iotagent-json#827)
+- Fix: allow send multiple measures to CB in a batch (POST /v2/op/update) and sorted by TimeInstant when possible, instead of using multiples single request (iotagent-json#825)
+- Fix: default express limit to 1Mb instead default 100Kb and allow change it throught a conf env var 'IOTA_EXPRESS_LIMIT' (iotagent-json#827)
```

Comment:
Issue https://github.com/telefonicaid/iotagent-node-lib/issues/1612 should be mentioned in the CNR of this PR.

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1607/commits/d9d0c26fe2caed2e08136780e30204380273f2ad